packages/paste-core/components/progress-steps/src/ProgressStepContent.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, type BoxProps } from "@twilio-paste/box";
+import type { Space } from "@twilio-paste/style-props";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+import React from "react";
+
+export interface ProgressStepContentProps extends HTMLPasteProps<"div"> {
+  children?: React.ReactNode;
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   *
+   * @default 'PROGRESS_STEP_CONTENT'
+   * @type {BoxProps['element']}
+   * @memberof ProgressStepContentProps
+   */
+  element?: BoxProps["element"];
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   *
+   * @default 'PROGRESS_STEP_CONTENT'
+   * @type {BoxProps['element']}
+   * @memberof ProgressStepContentProps
+   */
```

Comment:
update

Reply:
was adding this prop a design requirement?

Reply:
will update the code comment and yeah Sarah and Roxana asked for this prop